pkg/cache/queue_info_test.go

```diff
@@ -24,17 +24,20 @@ import (
 
 	"github.com/apache/incubator-yunikorn-core/pkg/common/configs"
 	"github.com/apache/incubator-yunikorn-core/pkg/common/resources"
+	"gotest.tools/assert"
 )
 
 // create the root queue, base for all testing
-func createRootQueue() (*QueueInfo, error) {
+func createRootQueue(t *testing.T) *QueueInfo {
```

Comment:
Same as before, I don't think it is a good idea to check the nil error inside of such helper functions. Unless you can print the trace correctly. Such as: https://github.com/apache/incubator-yunikorn-core/blob/366b635676e86e250d759f234fe1bfa44e4322c5/pkg/scheduler/tests/mock_rm_callback.go#L116

Reply:
Too complex and too much overhead, just leave it as is.

Reply:
This will also require reversing variable creation and checks for shadowing of `err`